packages/core/utils/src/replaceBundleReferences.js

```diff
@@ -32,117 +32,110 @@ export async function replaceBundleReferences({
   bundleGraph,
   contents,
   map,
```

Comment:
Can't leave a comment on code that was folded, but I was wondering if the comment describing this function needs to be updated at all since it seems like it changed a bit.